docs/standard/components.md

```diff
@@ -54,27 +53,34 @@ Mono is the runtime used by Xamarin apps.  It implements the .NET Standard, whic
 
 To learn more about Mono, see the [Mono documentation](http://www.mono-project.com/docs/).
 
+## .NET runtimes
+
+A runtime is the execution environment for a managed program. The OS is part of the runtime environment but is not part of the .NET runtime. Here are some examples of .NET runtimes:
+ 
+ - Common Language Runtime (CLR) for .NET Framework
+ - Core Common Language Runtime (CoreCLR) for .NET Core
+ - .NET Native for Universal Windows Platform 
+
 ## .NET tooling and common infrastructure
 
-Tooling for .NET is also common across each implementation of .NET.  These include, but are not limited to:
+Some tooling for .NET is common across all implementations of .NET.  These include, but are not limited to:
 
-* The .NET languages and their compilers
-* Runtime components, such as the JIT and Garbage Collector
-* .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
-* MSBuild, the build engine used to build projects
-* NuGet, Microsoft's package manager for .NET
-* The .NET Core CLI, a cross-platform command-line interface for building .NET projects
-* Open Source build orchestration tools, such as CAKE and FAKE
+- The .NET languages and their compilers
+- .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
+- MSBuild, the build engine used to build projects
+- NuGet, Microsoft's package manager for .NET
+- Open Source build orchestration tools, such as CAKE and FAKE
 
-The main takeaway here should be that there is a vast amount of tooling and infrastructure which is common for any "flavor" of .NET you choose to build your apps with.
+The main takeaway here should be that there is a vast amount of tooling and infrastructure which is common for any implementation of .NET you choose to build your apps with.
```

Comment:
"here" is unnecessary. Suggest a definitive statement; the "takeaway here" is conversational but wordy. "a vast amount of X and Y" seems vague and awkward. "which" :point_right: "that" ... and the prep is avoidable if u like.

> You have access to an extensive set of tools and infrastructure components that are available for any implementation of .NET you employ to build your apps and libraries.

mmmmm .... ahhhhh lol